src/frontend/src/utils/userNumber.ts

```diff
@@ -135,40 +139,15 @@ const readAnchors = (): Anchors => {
     item = {};
   }
 
-  if (typeof item !== "object") {
-    console.warn("bad identity", item, "type is", typeof item);
+  // Actually parse the JSON object
+  const parsed = Anchors.safeParse(item);
+  if (parsed.success !== true) {
+    const message = `ignoring malformed localstorage data: ` + parsed.error;
```

Comment:
not sure "ignoring malformed localstorage data" gonna be meaninungful for a random user